pages.fr/common/fzf.md

```diff
@@ -0,0 +1,29 @@
+# fzf
+
+> Recherche approximative en ligne de commande.
+> Similaire à `sk`.
+> Plus d'informations : <https://github.com/junegunn/fzf>.
+
+- Lance `fzf` sur tous les fichiers du répertoire spécifié :
+
+`find {{chemin/vers/répertoire}} -type f | fzf`
+
+- Démarre `fzf` pour les processus en cours :
+
+`ps aux | fzf`
+
+- Sélectionne plusieurs fichiers avec `Shift + Tab` et écrit dans un fichier :
+
+`find {{chemin/vers/répertoire}} -type f | fzf --multi > {{chemin/vers/fichier}}`
+
+- Lance `fzf` avec une requête spécifiée :
```

Comment:
Not sure if better... :shrug: 

> - Lance `fzf` avec une requête donnée :

Reply:
Yeah sounds more natural.

Oauis ca sonne plus naturel.